src/components/VideoParent/VideoPlayer/index.tsx

```diff
@@ -125,148 +145,142 @@ class VideoPlayer extends React.Component<Props, State> {
     this.video.current.webkitEnterFullscreen();
   };
 
-  setShowControls = (e:any) => { 
+  setShowControls = (e: any) => {
+    this.setState({
+      showControls: !this.state.showControls
```

Comment:
use updater in here.
[setstate](https://reactjs.org/docs/react-component.html#setstate)